examples/backend/src/domain/models/oauth-user-info.model.ts

```diff
@@ -0,0 +1,16 @@
+import { OAuthUser } from '@mobilejazz/harmony-nest';
+
+import { UserRole } from '../../data/entities/user.entity';
+
+export class OAuthUserInfoModel implements OAuthUser {
+    constructor(
+        readonly id: number,
+        readonly passwordSalt: string,
+        readonly passwordHashAlgorithm: string,
+        readonly role: UserRole,
+    ) {}
+
+    oauthId(): string {
```

Comment:
Not really, the interface method is just weirdly named, maybe we should open an issue to rename this to `getOAuthID()`. Changing this might break some project code (which should be easy to fix). **We could do it in this PR though**.

In any case, explicit access modifier added: 232ddf2e0f232ba5bda2427386005d8d3adad7e2

Reply:
@vilanovi do you remember what you where thinking here? Was your idea to use this as a method or as a property?

What Oriols first comment says is that if we add the `get` keyword then this can be used as `instance.oauthId` instead of `instance.oauthId()`. It's just sugar.

I'm leaned towards defining it as a method, as it's mostly for internal consumption by the harmony-nest package (I think).

Reply:
Both approaches work. Probably, this being only a read-only method, we could call it getOAuthId. Your call.

Reply:
`OAuthUserInfo` interface changed: 9fe20b3c1839821ebb2d682b66513b59245871e9